content/en/docs/reference/config/analysis/ist0137/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: DeploymentConflictingPorts
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: n/a
+---
+
+This message occurs when two services selecting the same workload with same target port but different ports.
+
+## An example
+
+Consider an Istio mesh with the following services:
+
+{{< text yaml >}}
+apiVersion: v1
```

Comment:
Nit: This will be more readable if the example is as **short** as possible.  Looking at https://kubernetes.io/docs/concepts/services-networking/service/ the examples are very small, e.g.

``` yaml
apiVersion: v1
kind: Service
metadata:
  name: my-service
spec:
  ports:
    - protocol: TCP
      port: 80
      targetPort: 9376
```

Can you make your example really small?